src/d3d11/d3d11_texture.cpp

```diff
@@ -562,15 +562,28 @@ namespace dxvk {
     if (imageFormatInfo(pImageInfo->format)->flags.test(DxvkFormatFlag::MultiPlane))
       return D3D11_COMMON_TEXTURE_MAP_MODE_BUFFER;
 
-    // Images that can be read by the host should be mapped directly in
-    // order to avoid expensive synchronization with the GPU. This does
-    // however require linear tiling, which may not be supported for all
-    // combinations of image parameters.
-    // HACK: Never use direct mapping; it doesn't work right with MoltenVK
-    // because we don't unmap memory.
-    //return this->CheckImageSupport(pImageInfo, VK_IMAGE_TILING_LINEAR)
-    //  ? D3D11_COMMON_TEXTURE_MAP_MODE_DIRECT
-    //  : D3D11_COMMON_TEXTURE_MAP_MODE_BUFFER;
```

Comment:
Verify whether this is still required or if new the new changes make this obsolete.

Reply:
Borderlands 3 produces the same output with these changes as with the CrossOver hack in place.

Reply:
SWJFO seems to work. Looks like the upsteam changes have made this hack obsolete.